tests/test_quantum_gradients.py

```diff
@@ -396,7 +396,9 @@ def circuit(x,y,z):
             autograd_val = grad_fn(*angle_inputs)
             for idx in range(3):
                 onehot_idx = eye[idx]
-                manualgrad_val = (circuit(angle_inputs + np.pi / 2 * onehot_idx) - circuit(angle_inputs - np.pi / 2 * onehot_idx)) / 2
+                param1 = angle_inputs + np.pi / 2 * onehot_idx
+                param2 = angle_inputs - np.pi / 2 * onehot_idx
+                manualgrad_val = (circuit(*param1) - circuit(*param2)) / 2
```

Comment:
how come the arguments are now dereferenced?

Reply:
The circuit QNode is defined as `circuit(x, y, z)`, so the arguments should be unpacked before passing. This is similar to the comment above - the existing test _should_ have been failing, not sure why it was passing. The above change allows it to pass with both caching on and off.